### PR TITLE
[15.0] [account_payment_widget_amount] - Fix account payment widget amount

### PR DESCRIPTION
**Pull request by hieulucky111**
When Invoice has Payment Term with Multiple Payment period. Partial amount will be wrong in some cases

### CODE REVIEW

File: account_payment_widget_amount/models/account_move_line.py [hunk 1]
```diff
@@ -1,5 +1,3 @@
-# Copyright 2018-2021 ForgeFlow S.L.
```
Comment (ChrisOForgeFlow) [inline account_payment_widget_amount/models/account_move_line.py]: You should not remove copyright section

**Review by ChrisOForgeFlow — COMMENTED** [review]
Can you add please test case, to understand cases when algorithm goes wrong?

File: account_payment_widget_amount/models/account_move_line.py (hunk 1 above)
Comment (hieulucky111) [reply account_payment_widget_amount/models/account_move_line.py]: Ok, i am working on some new test cases.

**Review by ChrisOForgeFlow — APPROVED** [review]
LGTM+

**Pull request closed without merging**